docs/contributing.md

```diff
@@ -41,9 +41,10 @@ The full report can be seen by opening `./coverage/lcov-report/index.html`.
 
 ## Releasing
 
-To release, go to the master branch and then run:
+To release, go to the master branch and then create a new branch with the version number:
```

Comment:
```suggestion
To release, go to the master branch and then create a new branch with the version number (minus the `v` prefix of the version, otherwise lerna fails to push tags):
```